cmd/snap/cmd_snap_op.go

```diff
@@ -599,7 +599,8 @@ type cmdRefresh struct {
 
 	Revision         string `long:"revision"`
 	List             bool   `long:"list"`
-	Time             bool   `long:"time"`
+	Time             bool   `long:"time" hidden:"yes"`
+	Timer            bool   `long:"timer"`
```

Comment:
Looking at the PR, there's a gut feeling that this is error prone. The difference between "time" and "timer" is too subtle, and people will end up using one or the other without realizing, and getting weird behavior. I suggest just introducing the new behavior, and only displaying schedule if one is indeed set and no timer is set.

There's a very small chance that this will break someone that was trying to get the old schedule even when nothing was set, but this risk is one-off and better than the risk of having error prone behavior forever.